xbmc/Application.cpp

```diff
@@ -1831,7 +1831,7 @@ bool CApplication::OnAction(const CAction &action)
       if (!m_appPlayer.IsPaused() && m_appPlayer.GetPlaySpeed() != 1)
         m_appPlayer.SetPlaySpeed(1);
 
-      CGUIComponent *gui = CServiceBroker::GetGUI();
+      IGUIComponent *gui = CServiceBroker::GetGUI().get();
```

Comment:
@Memphiz now that gui is a shared pointer, clients should check the shared pointer for being not null before using it.

Reply:
Afais they already did before...

Reply:
i stand corrected - just missed over 1000 unguarded gui usages ... ;)